native-code/android/index.md

```diff
@@ -77,6 +77,33 @@ For instructions on how to build and run, see
 [webrtc/examples/androidapp/README][3].
 
 
+### Using Android Studio
+
+  1. Build the project normally:
+
+     ~~~~~ bash
+     ninja -C out/Debug AppRTCMobile
```

Comment:
I used the same directory as above in the document in section "Compiling": https://webrtc.org/native-code/android/